packages/calypso-e2e/src/lib/pages/posts-page.ts

```diff
@@ -40,11 +39,13 @@ export class PostsPage {
 	 * Opens the Posts page.
 	 *
 	 * Example {@link https://wordpress.com/posts}
+	 * Example {@link https://wordpress.com/posts/usersiteslug.wordpress.com}
 	 */
-	async visit(): Promise< Response | null > {
-		const response = await this.page.goto( getCalypsoURL( 'posts' ) );
-		await this.waitUntilLoaded();
-		return response;
+	async visit( { siteSlug, timeout }: { siteSlug?: string; timeout?: number } = {} ) {
+		const url = `posts/${ siteSlug }`;
+		await this.page.goto( getCalypsoURL( url ), { timeout: timeout } );
+
+		await this.waitUntilLoaded( timeout );
```

Comment:
Same changes as `PagesPage`, but retaining the existing `waitUntilLoaded` method for compatibility.